server/request_handler.go

```diff
@@ -101,6 +102,13 @@ func (r *RpcRequestHandler) process() {
 	// Parse JSON RPC payload
 	var jsonReq *types.JsonRpcRequest
 	if err = json.Unmarshal(body, &jsonReq); err != nil {
+		r.requestRecord.UpdateRequestEntry(r.req, http.StatusBadRequest, err.Error())
```

Comment:
every single batch request will show up as error in the database. is this actually helpful?

Reply:
lets remove for now